### PR TITLE
 [Issue #93] finished implementing archive project

### PR DESCRIPTION
**Pull request by tosinoni**

### CODE REVIEW

File: src/main/resources/static/js/directives/kapi-projects-table/kapi-projects-table.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     </thead>
     <tbody>
         <tr ng-click="projectClicked(row)" ng-repeat="row in data">
-            <td>{{row.id}}</td>
+            <td><i ng-class="row.isArchive ? 'icon-pin' : ''"> </i>{{row.id}}</td>
```
Comment (mohameddahrouj) [inline src/main/resources/static/js/directives/kapi-projects-table/kapi-projects-table.html]: This is nice 👍🏻

File: src/main/resources/static/js/projects/project-controller.js [hunk 2]
```diff
@@ -13,6 +13,9 @@ angular.module('CapstoneSystem')
             if(project) {
                 $scope.project = project;
 
+                $scope.isArchive = !project.isArchive;
+                $scope.isUnArchive = project.isArchive;
+
```
Comment (mohameddahrouj) [inline src/main/resources/static/js/projects/project-controller.js]: Probably could have used one variable but it’s ok

File: src/main/resources/static/views/projects/project.html [hunk 3]
```diff
@@ -17,7 +17,8 @@ <h1 class="portalheader">
                         <div ng-cloak ng-if="isProjectForProf">
                             <button title="Edit" ng-click="openEditProjectModal()" type="button" class="btn btn-warning"><i class="icon-pencil"></i> Edit</button>
                             <button ng-click="openAddStudentToProjectModal()" title="Students" type="button" class="btn btn-primary"><i class="icon-plus"></i> Students</button>
-                            <button title="Archive" type="button" class="btn btn-success"><i class="icon-pin"></i> Archive</button>
+                            <button ng-cloak ng-if="isArchive" ng-click="archiveProject()" title="Archive" type="button" class="btn btn-success"><i class="icon-pin"></i> Archive</button>
+                            <button ng-cloak ng-if="isUnArchive" ng-click="unArchiveProject()" title="unArchive" type="button" class="btn btn-success"><i class="icon-pin"></i> UnArchive</button>
```
Comment (mohameddahrouj) [inline src/main/resources/static/views/projects/project.html]: Nice and dynamic 👌🏽

File: src/main/resources/static/js/projects/project-controller.js (hunk 2 above)
Comment (tosinoni) [reply src/main/resources/static/js/projects/project-controller.js]: Yeah true. but i realized it would still be the same thing.

**Pull request merged**